vividus-plugin-db/src/test/java/org/vividus/bdd/steps/db/DatabaseStepsTests.java

```diff
@@ -155,9 +152,9 @@ void beforeEach()
     }
 
     @Test
-    void testExecuteSql() throws SQLException
+    void testExecuteSql()
     {
-        mockDataSource(QUERY, DB_KEY, mockResultSet(COL1, VAL1));
+        mockQueryForList(QUERY, DB_KEY, Collections.singletonList(Map.of(COL1, VAL1)));
```

Comment:
```suggestion
        mockQueryForList(QUERY, DB_KEY, List.of(Map.of(COL1, VAL1)));
```

Reply:
the same for all other occurrences

Reply:
Ok. Could you please explain why here List.of is profitable?